src/InputWithLabel.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function InputWithLabel ({todoTitle, handlerTitleChange, children}){
+    const inputRef = React.useRef();
+    React.useEffect(() =>{
+            inputRef.current.focus();
+    });
+    return(
+        <>
+            <label htmlFor='todoTitle'>{children} </label>
+            <input 
+            ref={inputRef}
+            type="text" 
+            id='todoTitle' 
+            name='title' 
```

Comment:
Consider passing all the attributes for the input element as `props`. This strategy can improve the reusability of your `nputWithLabel component.